libs/sdk/prettier-schematics/package.json

```diff
@@ -24,9 +24,9 @@
     "migrations": "./src/schematics/migrations/migration-collection.json",
     "packageGroup": {
       "@skyux-sdk/prettier-schematics": "0.0.0-PLACEHOLDER",
-      "@trivago/prettier-plugin-sort-imports": "^4.3.0",
-      "eslint-config-prettier": "^9.1.0",
-      "prettier": "^3.2.5"
+      "@trivago/prettier-plugin-sort-imports": "^5.2.2",
```

Comment:
Tested this on a few SPAs; version 5 (and version 10, below) do not have breaking changes that will affect our consumers.

ESLint's flat config requires `eslint-config-prettier@10`; version 10 also works for `.eslintrc.json` files as well.